src/main/java/com/google/ehub/data/CommentData.java

```diff
@@ -8,11 +8,17 @@
   private final String comment;
   private final Long timestampMillis;
   private final String username;
+  private final Long commentId;
+  private final String commentKind;
```

Comment:
do you need `commentKind`? I believe` commentId` should be enough.

commentKind will always be "comment" so i don't think we need to store it. We can just use the constant COMMENT_KIND_KEY